src/main/java/io/github/cottonmc/resources/tag/WorldTagReloadListener.java

```diff
@@ -0,0 +1,47 @@
+package io.github.cottonmc.resources.tag;
+
+import java.util.Map;
+
+import com.google.common.util.concurrent.MoreExecutors;
+
+import io.github.cottonmc.resources.CottonResources;
+import net.fabricmc.fabric.api.resource.IdentifiableResourceReloadListener;
+import net.minecraft.resource.ResourceManager;
+import net.minecraft.resource.SynchronousResourceReloadListener;
+import net.minecraft.tag.RegistryTagContainer;
+import net.minecraft.tag.Tag;
+import net.minecraft.tag.TagContainer;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+public class WorldTagReloadListener implements SynchronousResourceReloadListener, IdentifiableResourceReloadListener {
```

Comment:
Why not just implement SimpleSynchronousResourceReloadListener?

Reply:
Didn't know it existed. Fixed!